### ISSUE TITLE
Large MERRA2 climatology files copied for many regression tests, even though they are not neeeded

### ISSUE THREAD

**Issue by climbfuji** [ufs-community/ufs-weather-model#528]
## Description

Several regression test setup scripts in `tests/fv3_conf` are copying the new MERRA2 aerosol climatology files without checking if the run actually needs it (flag `iaerclm`). Since many of these files are large, we should avoid copying them when not needed.

Example from `tests//fv3_conf/gfs_v16_run.IN`:
```
...
cp    @[INPUTDATA_ROOT]/FV3_input_data/diag_table_gfdlmp diag_table
cp    @[INPUTDATA_ROOT]/FV3_input_data_gsd/field_table_suite2 field_table
cp    @[INPUTDATA_ROOT]/FV3_input_data/*configure .
for n in 01 02 03 04 05 06 07 08 09 10 11 12; do
cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m${n}.nc aeroclim.m${n}.nc
done
cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_BC.v1_3.dat  optics_BC.dat
cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_OC.v1_3.dat  optics_OC.dat
cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_DU.v15_3.dat optics_DU.dat
cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_SS.v3_3.dat  optics_SS.dat
cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_SU.v1_3.dat  optics_SU.dat
...
```

And no, the old argument to just link the files is not valid - the run directories need to be fully self-contained, but not contain unnecessary large data sets.

### To Reproduce:

Run the regression tests and look at how many directories contain those files.

## Output
File sizes in each run directory:
```
...
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:28 aeroclim.m01.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:28 aeroclim.m02.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:28 aeroclim.m03.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:28 aeroclim.m04.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m05.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m06.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m07.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m08.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m09.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m10.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m11.nc
-rwxr-xr-x  1 Dom.Heinzeller esrl 972M Apr 15 12:29 aeroclim.m12.nc
...
-rw-r--r--  1 Dom.Heinzeller esrl 198K Apr 15 12:29 optics_BC.dat
-rw-r--r--  1 Dom.Heinzeller esrl 451K Apr 15 12:29 optics_DU.dat
-rw-r--r--  1 Dom.Heinzeller esrl 198K Apr 15 12:29 optics_OC.dat
-rw-r--r--  1 Dom.Heinzeller esrl 491K Apr 15 12:29 optics_SS.dat
-rw-r--r--  1 Dom.Heinzeller esrl 100K Apr 15 12:29 optics_SU.dat
...
```

**Comment by junwang-noaa** [comment]
@climbfuji This is fixed in PR#561, let me know if you still see the issue.

**Comment by climbfuji** [comment]
> @climbfuji This is fixed in PR#561, let me know if you still see the issue.

Thanks, I saw that yesterday when reviewing the PR. This will solve the issue.
